daemon/daemon_windows.go

```diff
@@ -221,11 +220,7 @@ func verifyDaemonSettings(config *config.Config) error {
 func checkSystem() error {
 	// Validate the OS version. Note that docker.exe must be manifested for this
 	// call to return the correct version.
-	osv := system.GetOSVersion()
-	if osv.MajorVersion < 10 {
```

Comment:
I think this check was redundant (as we check build below), but let me know if this would still be needed

Reply:
There's no harm keeping it. It depends entirely if there ever was a version 8.x where x > 17763, but that to the best of my knowledge isn't true as build numbers have been increasing since NT days.

Reply:
Added it back 👍